### PR TITLE
Fixing sync interrupt syncstatus sendprogress

### PR DESCRIPTION
**Pull request by juanky201271**
* new version of `zingolib` with the new field: `sync_interrupt` in `syncstatus` & `sendprogress`.
* timeout from 2 sec to 5 sec -> updating the UI with info of syncing in `Report syncing screen`.
* timeout from 60 sec to 30 sec -> asking for new blocks to sync. Sometimes wait 1 minute for start syncing new blocks is a lot of time.
* timeout 2 sec -> sending it's ok. I ask for this new field: `sync_interrupt` and if the value is `false`, I set it to `true` because meanwhile the App is sending I want this value always to `true`. (edge case: when you are in the black screen for sending, maybe you can put the App in background or some notification come up... no matter what happen I need `true` on it all the time while sending).

I'm still doing testing with this.

### CODE REVIEW

**Comment by juanky201271** [comment]
I'm working with the Report Screen, and I fixed the problem report for Pacu about the birthday block, and I fixed. Now is the same that the wallet.

**Comment by juanky201271** [comment]
IOS - Testflight - v0.1.1 (119) Zingo! Test available for testers. WIth the new version of `zingolib` with `sync_interrupt`.

**Comment by juanky201271** [comment]
Android - v0.1.1 (118) Zingo! Test available. WIth the new version of zingolib with sync_interrupt.

**Comment by juanky201271** [comment]
Android - v0.1.1 (119) Zingo! Test available. WIth the new version of zingolib with sync_interrupt.

**Comment by juanky201271** [comment]
Hot fix:
* the new header is working with LoadingApp component.
* only one server is working.
* settings working with only one server.

**Comment by juanky201271** [comment]
IOS - Testflight - v0.1.1 (120) Zingo! Test available for testers, with a couple of Hot fixes.

File: app/BackgroundSync.ts [hunk 1]
```diff
@@ -62,7 +62,7 @@ const BackgroundSync = async (task_data: any) => {
         // update batch_num with the new value, otherwise never change
         batch_num = ss.batch_num;
       }
-    }, 2000);
+    }, 5000);
```
Comment (zancas) [inline app/BackgroundSync.ts:65]: Is this in milliseconds?

Comment (juanky201271) [reply app/BackgroundSync.ts:65]: Yes, this only affect to the Sync Report Screen... you can see the info every 5 seconds, and I included a spin circle with 5 steps... IMO looks good, and the App response is better, plus Matilda doesn't care about this screen. I think you can see this effect in v120...

File: app/rpc/RPC.ts [hunk 2]
```diff
@@ -1075,6 +1075,12 @@ export default class RPC {
         const progress = await JSON.parse(pro);
         const sendId = progress.id;
 
+        // bacause I don't know what the user are doing, I force every 2 seconds
+        // the interrupt flag to true if the sync_interrupt is false
+        if (!progress.sync_interrupt) {
+          await RPC.rpc_setInterruptSyncAfterBatch('true');
```
Comment (zancas) [inline app/rpc/RPC.ts:1081]: Wow!

**Review by zancas — COMMENTED** [review]
Looks Good To Me (LGTM). I would like this PR to include an E2E test.

File: app/rpc/RPC.ts (hunk 2 above)
Comment (juanky201271) [reply app/rpc/RPC.ts:1081]: Maybe never happen, but if this flag become `false`, I will change to `true` right away... (only in the time window when the App is sending)

**Comment by fluidvanadium** [comment]
works as ZingoTest on my android: consistent sync

**Review by zancas — APPROVED** [review]
Based on FluidV's review comment, I will approve.

**Pull request merged**